src/System.Private.CoreLib/shared/System/Reflection/Emit/Opcode.cs

```diff
@@ -110,7 +111,7 @@ public short Value
 
         private static volatile string[] g_nameCache;
```

Comment:
I'd expect this to be `string[]?`.  The compiler doesn't currently investigate statics, unfortunately.

Reply:
> The compiler doesn't currently investigate statics, unfortunately.

Good to know, thanks

> I'd expect this to be string[]?

For private variable is that necessary?